Added extra documentation for Windows Audit Policies in Syscheck whodata (#5676)
wazuh/wazuh-documentation: source/user-manual/capabilities/auditing-whodata/who-windows-policies.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::
  :description: Learn more about the manual configuration of the Local Audit Policies and SACLs in Windows in this section of the Wazuh documentation.

.. _who-windows-policies:

Manual configuration of the Windows Audit Policies
==================================================

Local Audit Policies in Windows

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Author: Done

    gpedit.msc

Go to **Security Settings** -> **Advanced Audit Policy Configuration** and configure to ``Success`` the following policies:

1) **Object Access** -> **Audit File System**
2) **Object Access** -> **Audit Handle Manipulation**

.. thumbnail:: ../../../images/whodata/audit-policies.png
    :title: Audit policies configuration
    :align: center
    :width: 60%

> If your system doesn't allow configuring subcategories through *Advanced Audit Policy Configuration*, you can configure to *Success* the entire category. These policies are part of the *Audit object access* category. Go to to **Security Settings** -> **Local Policies** -> **Audit Policy** and configure this category.

Reviewer: ```suggestion
If your system doesn't allow configuring subcategories through *Advanced Audit Policy Configuration*, you can configure to *Success* the entire category. These policies are part of the *Audit object access* category. Go to **Security Settings** -> **Local Policies** -> **Audit Policy** and configure this category.
```

.. thumbnail:: ../../../images/whodata/audit-policies-old-method.png
    :title: Audit policies configuration (old method)
    :align: center
    :width: 60%

SACLs in Windows

> ^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
----------------
```

    gpedit.msc

> Go to ``Security Settings -> Advanced Audit Policy Configuration`` and configure to Success the following policies:

Author: Done

> 1) Object Access -> Audit File System

Author: Done

> 2) Object Access -> Audit Handle Manipulation

Author: Done

.. thumbnail:: ../../../images/whodata/audit-policies.png
    :title: Audit policies configuration
    :align: center
    :width: 60%

> These policies are part of the Audit Object Access category, so another possibility is to configure the entire category to Success in ``Security Settings -> Local Policies -> Audit Policy``,

Author: Done

but this is only recommended in systems where the subcategories cannot be configured through Advanced Audit Policy Configuration:

.. thumbnail:: ../../../images/whodata/audit-policies-old-method.png
    :title: Audit policies configuration (old method)
    :align: center
    :width: 60%

SACLs in Windows
^^^^^^^^^^^^^^^^

A system access control list (SACL) enables administrators to log attempts to access a secured object.

> SACLs of each directory can be checked and modified through its Properties, Security tab, and clicking on Advanced:

Reviewer: ```suggestion
You can check and modify SACLs of each directory through **Properties**, selecting the **Security** tab, and clicking on **Advanced**:
```

.. thumbnail:: ../../../images/whodata/windows-security-setting.png
    :title: Windows advanced security settings
    :align: center
    :width: 60%

> In the Auditing tab it is necessary to have a Success entry:

Reviewer: ```suggestion
In the Auditing tab it's necessary to have a *Success* entry:
```

Author: Done

.. thumbnail:: ../../../images/whodata/sacl-configuration.png
    :title: Auditing entry success
    :align: center
    :width: 60%

> It can be created by adding these advanced permissions:

Reviewer: ```suggestion
You can create it by adding these advanced permissions:
```

Author: Done
